include/cppkafka/kafka_handle_base.h

```diff
@@ -217,9 +217,9 @@ class CPPKAFKA_API KafkaHandleBase {
     std::vector<GroupInformation> fetch_consumer_groups(const char* name);
     void save_topic_config(const std::string& topic_name, TopicConfiguration config);
 
+    Configuration config_;
     HandlePtr handle_;
```

Comment:
Good catch. Could you actually move the handle all the way down? That should be the first thing that gets destructed. I'm concerned about anything in `TopicConfigurationMap` having the same issue as the one you're describing.

Reply:
Yep, good suggestion. I force pushed the patch.